lib/widget/shared/alert_style.dart

```diff
@@ -17,3 +17,36 @@ AlertStyle alertStyle = AlertStyle(
   ),
   alertAlignment: Alignment.center,
 );
+
+class Dialogs {
+  static Future<void> alertDialog(BuildContext context) async {
+    await showDialog(
+      context: context,
+      builder: (context) {
+        return AlertDialog(
+            shape:
+                RoundedRectangleBorder(borderRadius: BorderRadius.circular(15)),
+            title: Text("กรุณาทำแบบประเมินให้ครบถ้วน",
+                style: Theme.of(context).textTheme.bodyText2,
+                textAlign: TextAlign.center),
+            actions: [
+              FlatButton(
+                color: Color(0xFFC37447),
+                child: Container(
+                  width: MediaQuery.of(context).size.width,
+                  child: Center(
+                    child: Text(
+                      "ตกลง",
+                      style: TextStyle(color: Colors.white, fontSize: 16),
+                    ),
+                  ),
+                ),
+                onPressed: () {
+                  Navigator.pop(context);
+                },
+              )
+            ]);
+      },
+    );
```

Comment:
If we want to refactor all `Dialogs` thing into like this one. So, we need to specify what is this dialog for. Since it has more than one type of dialog.

Reply:
I change the alert dialog name same as web 
879a057